Cargo.toml

```diff
@@ -4,13 +4,17 @@ name = "html5ever"
 version = "0.0.0"
 authors = [ "The html5ever Project Developers" ]
 
+[lib]
+name = "html5ever"
+
+# https://github.com/rust-lang/cargo/issues/1512
+doctest = false
+
 [dependencies]
 time = "0"
 log = "0"
-
-# servo/string-cache#80
-phf = "0.6"
-phf_macros = "0.6"
+phf = "0.7"
+phf_macros = "0.7"
```

Comment:
Semver says anything goes before 1.0, but Cargo’s extrapolation of it is that any 0.7.x should be compatible with 0.7.y with y <= x, while 0.8 is not. So if phf follows that convention, 0.8 will probably break html5ever while (in a post-Rust-1.0 world) 0.7 will still be available and compatible.

That said, I’ll make them `= "0"` if you still feel it’s the right thing to do.